internal/handlers/handler_test.go

```diff
@@ -102,6 +101,7 @@ func handle(ctx context.Context, t *testing.T, handler *pg.Handler, req *types.D
 
 func TestFind(t *testing.T) {
 	t.Parallel()
+
```

Comment:
No, we don't have an empty line there; check out other tests.

(and we are going to delete all those tests anyway)

Reply:
removed